Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG RUBY_VERSION=2.6.6
+ARG BUNDLER_VERSION=2.1.4
+
+# Defining ruby version
+FROM ruby:$RUBY_VERSION
+
+# Set working dir and copy app
+WORKDIR /usr/src/app
+COPY . .
+
+# Prerequisites for gems install
+RUN apt-get install tzdata \
+                    git
+
+# Install bundler and gems
+RUN gem install bundler:$BUNDLER_VERSION
+RUN bundle install
+
+# Params
+ARG RAILS_ENV="production"
+ARG RAILS_SERVE_STATIC_FILES="true"
+ARG RELATIVE_URL_ROOT="/app/ppd"
+ARG API_SERVICE_URL="http://localhost:8080"
```

Comment:
I'm confused by the API_SERVICE_URL defaulting to `localhost:8080`. That will only be useful if a developer happens to be running a development copy of Sapi-NT locally. I know  you are doing that at the moment, but that's not how we will usually work. I think it would be better to make the default URL be one that will "just work" if someone needs to make a quick change, and otherwise if we happen to be running Sapi-NT locally then we can pass in `localhost:8080` as an arg.